qldbdriver/qldbdriver_test.go

```diff
@@ -485,7 +483,7 @@ func TestExecuteWithRetryPolicy(t *testing.T) {
 		testDriver.qldbSession = mockSession
 
 		testDriver.sessionPool = make(chan *session, 10)
-		testDriver.semaphore = sync2.NewSemaphore(int(10), time.Duration(10)*time.Second)
+		testDriver.semaphore = makeSemaphore(int(10))
```

Comment:
I see both makeSemaphore(10) and makeSemaphore(int(10)). Can we get rid of the needless int(10) conversion?